src/platform/site-wide/sass/modules/_m-overrides.scss

```diff
@@ -0,0 +1,7 @@
+// Override path variables for imported frameworks
```

Comment:
made a separate overrides file so that these can be easily imported into the different site-wide csses